package com.desafio.exception;

public record ErrorResponse(String error, String message) {
}